### PR TITLE
Fix OpenFL configuration dialog

### PR DESCRIPTION
**Pull request by EBatTiVo**
@as3boyan Please review.

### CODE REVIEW

File: src/common/com/intellij/plugins/haxe/ide/projectStructure/ui/HaxeConfigurationEditor.form [hunk 1]
```diff
@@ -273,7 +273,7 @@
                           <text resource-bundle="com/intellij/plugins/haxe/HaxeBundle" key="haxe.openfl.arguments"/>
                         </properties>
                       </component>
-                      <component id="39470" class="com.intellij.ui.RawCommandLineEditor">
+                      <component id="39470" class="com.intellij.ui.RawCommandLineEditor" binding="myOpenFLArgumentTextField">
```
Comment (as3boyan) [inline src/common/com/intellij/plugins/haxe/ide/projectStructure/ui/HaxeConfigurationEditor.form]: So it was not connected before? `myOpenFLArgumentTextField` was just useless field? 

BTW, http://stackoverflow.com/questions/27865629/openfl-arguments-cant-be-added

**Comment by as3boyan** [comment]
Good catch.

**Comment by as3boyan** [comment]
Tested it, works fine.
About repaint error - I was able to reproduce it, but I'm not sure why it is going on.
Approved for merge.

File: src/common/com/intellij/plugins/haxe/ide/projectStructure/ui/HaxeConfigurationEditor.form (hunk 1 above)
Comment (EBatTiVo) [inline src/common/com/intellij/plugins/haxe/ide/projectStructure/ui/HaxeConfigurationEditor.form]: Correct.

Thanks for the link.  I'll go answer it after we release -- even though it's an old thread.

**Comment by EBatTiVo** [comment]
I believe the repaint error is because the OpenFL and NMML panels are the same size, so the automatic repaint didn't occur as it does with the other panels because Swing doesn't consider the area dirty unless there was a size change.  So, the app needed to trigger the paint itself.  The clue was if focus changed away to another app, the panel would redraw correctly.

**Pull request merged**